ARrehab/ARrehab/MovementGame.swift

```diff
@@ -47,15 +54,49 @@ class MovementGame : Minigame {
         self.total = num
         // For our purposes, we placed the player as a 2 centimeter sphere around the camera.
         self.playerCollisionEntity = TriggerVolume(shape: ShapeResource.generateSphere(radius: 0.01), filter: CollisionFilter(group:playerCollisionGroup, mask: targetCollisionGroup))
```

Comment:
This seems like a better collision method than generating a collision component (like we do in the TileCollider.addCollision() method —> Can you look into replicating this there?

Reply:
So the problem with a TriggerVolume is I can't extend it meaning we can't add any custom code beyond collision capabilities.

The Player class depends on extending those capabilities to include `onTile` and its `AnchoringComponent`